jest.config.ts

```
@@ -88,7 +88,14 @@ export default {
   // ],
 
   // A map from regular expressions to module names or to arrays of module names that allow to stub out resources with a single module
-  // moduleNameMapper: {},
+  moduleNameMapper: {
+    "^@airframes/ads-runtime-ts$":
```

ESLint

[error] Replace `"^`@airframes/ads-runtime-ts`$"` with `'^`@airframes/ads-runtime-ts`$'`

(prettier/prettier)